README.adoc

```diff
@@ -9,7 +9,7 @@ More info you can find in link:doc/overview.adoc[Overview document].
 . Install git if not installed already, install ruby 2.4+ if not installed
```

Comment:
Can we update this with latest version of ruby that is being used? Alternatively mention it somewhere in the document.

Reply:
@asood-rh I don't think we need to change that as `2.4+` indicated that the minimum version is 2.4 or above.

Reply:
@pruan-rht Would it work with ruby 3.0.0 as well? If that is supported then it does not need to change otherwise intent of comment is to clarify the exact version.

Reply:
@asood-rh I haven't tried it...it's better to just put the minimum instead of a specific version, otherwise we would need to keep updating the doc.

Reply:
@asood-rh I agree with @pruan-rht  here to keep minimum version. Adding different versions can be contentious and would need update to README. Even if we test it now against, lets say, 3.0.0, we may have to test again and update the compatibility. Not that its not a good idea, we can definitely make an effort to keep that up-to-date, but for the purpose of this PR I'd leave the version info as is.